system-variables.md

```diff
@@ -710,6 +710,31 @@ For a system upgraded to v5.0.0-rc from an earlier version, if you have not modi
 - Default value: 60
 - This variable is used to set the step of the Prometheus statement generated when querying `METRIC_SCHEMA`. The unit is second.
 
+## tidb_multi_statement_mode <span class="version-mark">New in v4.0.11</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: OFF
+- Permitted values: OFF, ON, WARN
+- To reduce the impact from SQL injection attacks, TiDB now prevents multiple queries being executed in the same `COM_QUERY` call by default.
+- This variable is intended to be used as part of an upgrade path from earlier versions of TiDB. The following behaviors apply:
+
+| Client setting         | `tidb_multi_statement_mode` value | Multiple statements permitted? |
+|------------------------|-----------------------------------|--------------------------------|
+| Multi Statements = ON  | OFF                               | Yes                            |
```

Comment:
Are `Multi Statements` and `Multiple statements` the same thing? If so, shall we unify the spelling?

Reply:
Kind of.

We can unify the spelling, but each client library has a different name for this. "Multi statements"  is used here to refer to the client setting, which is more a concept than a specific feature.